src/routing_filter.h

```diff
@@ -122,7 +122,7 @@ routing_filter_get_next_value(uint64 found_values, uint16 last_value)
 static inline bool
 routing_filter_is_value_found(uint64 found_values, uint16 value)
 {
-   return found_values & (1 << value);
+   return ((found_values & (1 << value)) != 0);
```

Comment:
@rtjohnso - Weirdly, this one test to do with filters is tripping due to above code. The old code would return any int value > 255 to mean `TRUE`'. Now, if `value` is 8, `(1 << value)` becomes 256, and so `found_values & (1 << value)` no longer fits in 1-byte unsigned char return type. 

This fn could incorrectly return `FALSE` in cases where it should have returned `TRUE`.

With this fix, my manual runs of `test.sh` are clean, so I am now going to rely on CI to ferret out bugs due to any other such wrong code.

Reply:
Good catch!